src/pixel-display-impl_.toit

```diff
@@ -77,6 +77,17 @@ abstract class PixelDisplay implements Window:
   inner-width: return driver_.width
   inner-height: return driver_.width
 
+  /**
+    By default, the display is rendered in patches that have
+      a max size of 2-8k, depending on the bits per pixel.  On devices
```

Comment:
```suggestion
      a max size of 2-8k pixels, depending on the bits per pixel.  On devices
```

Reply:
From a user's point of view setting the memory used might be easier. Like "you are allowed to use 20K of RAM for each patch". Not really sure about that.

Reply:
That's what it is already though.

Reply:
Why does the TwoColorPixelDisplay then set it to a higher value? I would have expected the 2-color display to use the least amount of memory.